README.md

```diff
@@ -66,6 +67,7 @@ The RAI framework aims to:
 - [x] Improved Human-Robot Interaction with voice and text.
 - [x] Additional tooling such as GroundingDino.
 - [x] Support for at least 3 different AI vendors.
+- [x] Debugging assistant for ROS 2.
```

Comment:
General question (probably better to discuss in an issue):
How do you envision the future of this tool?
Will it be standalone agent just for ros2 system debugging or should debugging agent be added to other rai agents like `state_based` agent?

Reply:
Standalone, however this is an interesting idea. I will add an issue for that